packages/pyyaml/meta.yaml

```diff
@@ -5,10 +5,12 @@ source:
   url: https://files.pythonhosted.org/packages/36/2b/61d51a2c4f25ef062ae3f74576b01638bebad5e045f747ff12643df63844/PyYAML-6.0.tar.gz
   sha256: 68fb519c14306fec9720a2a5b45bc9f0c8d1b9c72adf45c37baedfcd949c35a2
 build:
-  cflags:
-    "-I$(PYTHONINCLUDE)\n-I$(PYODIDE_ROOT)/emsdk/emsdk/fastcomp/emscripten/system/include/libc\n\
-    -I$(PYODIDE_ROOT)/packages/libyaml/build/include\n"
-  ldflags: "-L$(PYODIDE_ROOT)/packages/libyaml/build/\n"
+  cflags: |
+    -I$(PYTHONINCLUDE)
+    -I$(PYODIDE_ROOT)/emsdk/emsdk/fastcomp/emscripten/system/include/libc
```

Comment:
```suggestion
```
This directory hasn't existed since #1102 was merged in January 2021, but I guess we didn't notice.